src/aosm/azext_aosm/_configuration.py

```diff
@@ -31,14 +42,25 @@
         "Version of the artifact. For VHDs this must be in format A-B-C. "
         "For ARM templates this must be in format A.B.C"
     ),
+    "nfvi_site_name": "Name of the NFVI Site",
+    "nfvi_site_type": "Type of the NFVI Site",
+    "cg_schema_name": "Name of the CG Schema to be deployed. This defined the configuration required to deploy the NSD",
```

Comment:
Maybe somewhere write that this will make one and then you can manually add more schemas? Not sure about this but I know you can have multiple cgschemas so we shouldprobs specify that

Reply:
I can add that description now, but I suspect that we will do some work in the future to enable having more than one schema